src/log.js

```diff
@@ -78,9 +78,9 @@ genotet.logger.sendBack = function() {
   if (genotet.logger.logList.length) {
     var params = {
       logs: genotet.logger.logList,
-      username: 'anonymous' // will call api of user system
+      username: genotet.user.info.username
     };
-    $.post(genotet.data.logURL, params, 'json');
+    $.post(genotet.data.logUrl, params, 'json');
```

Comment:
Here we need to capture the post error. If the log post was not successful then we should retry the post some time later, rather than clean the logList below. Looks like a previous neglect sneaked in.